src/config.ts

```diff
@@ -121,4 +129,22 @@ CQIDAQAB
   public getPhoneNumberVerified(): boolean {
     return this.phoneNumberVerified;
   }
+
+  public getAuthCodeRequestParamsStore(): Record<
+    string,
+    AuthRequestParameters
+  > {
+    return this.authCodeRequestParamsStore;
+  }
```

Comment:
Do we want this, rather than retrieving a specific code?

Reply:
Yeah I'm not quite sure why I did this tbh - think I did this one first and changed my mind after 😆 getting one would be nicer